HomePage.txt

```diff
@@ -0,0 +1,18 @@
+package gui;
+
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebElement;
+
+import java.time.Duration;
+
+public class HomePage extends BasePage {
+
+
+     WebElement faUser = driver.findElement(By.xpath("//i[@class='fak fa-user text-xl text-gray-900 dark:text-white']"));
```

Comment:
It would be more convenient to use PageFactory and initialize this element as:
@FindBy(xpath = "//i[@class='fak fa-user text-xl text-gray-900 dark:text-white']")
private WebElement faUser;